### ISSUE TITLE
Feature reset and confirmation for lost data

### ISSUE THREAD

**Issue by iliaross** [virtualmin/virtualmin-gpl#774]
Jamie, if all features are set to reset without setting **Continue even if settings would be lost?** to **Yes** e.g.:

<img width="1440" alt="image" src="https://github.com/virtualmin/virtualmin-gpl/assets/4426533/d4680bd8-5909-4513-9e50-c66180049bbd">

Then the whole process seems to continue for each feature independently rather than showing first a consolidated message about which features' settings would be lost:

<img width="231" alt="image" src="https://github.com/virtualmin/virtualmin-gpl/assets/4426533/ef89e06c-9a62-4eac-83f3-d7dd6b0af81f">

It still shows the button at the bottom of the page, though it doesn't seem clear, as it's related to the DNS feature at the top of the list.

I think we should ask for the confirmation before proceeding, if any of the features on the list are going to be lost, in a consolidated message.

**Comment by jcameron** [comment]
Yeah it's not very consistent. I'll look into fixing this UI for the next release ...

**Comment by iliajie** [comment]
Since we're already reworking and fixing Reset Features throughout, I believe we should tackle this now and be done with it.

**Comment by iliajie** [comment]
We can wait, no problem. Though, if you don't have time to do it right now, and want me to send you a PR today, just let me know..

**Comment by iliajie** [comment]
> No let's wait ... this is a cleanup, not a critical fix.

Where are we with this one?

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
